Formula/o/odin.rb

```diff
@@ -56,7 +55,13 @@ def install
       "vendor/raylib/macos-arm64"
     end
 
-    ln_s Formula["glfw"].lib/"libglfw3.a", buildpath/"vendor/glfw/lib/darwin/libglfw3.a"
+    glfw_installpath = if OS.linux?
+      "vendor/glfw/lib"
+    else
+      "vendor/glfw/lib/darwin"
+    end
+
+    ln_s Formula["glfw"].lib/"libglfw3.a", buildpath/glfw_installpath/"libglfw3.a"
```

Comment:
Does this symlink not get broken once we install the `vendor` directory into the `prefix`?

Reply:
The test I added that passes `-define:GLFW_SHARED=false` should break if that is the case I think, that one would try to find and link with it. I also tried it locally and it seems to work.

Let me know if there's something else you want to see here or if there's something else I could test to make sure.